package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "oas-normalize",
   "version": "7.1.1",
-  "description": "OpenAPI 3.x or Swagger 2.0? YAML or JSON? URL, path, string or object? Who cares! It just works.",
+  "description": "Tooling for converting, valiating, and parsing OpenAPI, Swagger, and Postman API definitions",
```

Comment:
Not totally sold on this new description but I've always loathed the language in the current one.

Reply:
Maybe "Tooling for converting, normalizing, and validating API definitions."?

Reply:
Not sure how keywords and searching work on npm but I like the shorter description if that means it's still discoverable on npm using the terms OpenAPI/Swagger/Postman